### PR TITLE
WIP: prerequisites doc

### PR DESCRIPTION
**Pull request by anvddriesch**
This sums up what we did today in order to create access for the CAPA controller.

### CODE REVIEW

**Comment by TheoBrigitte** [comment]
Towards: https://github.com/giantswarm/giantswarm/issues/16112

File: README.md [hunk 1]
```diff
@@ -2,3 +2,71 @@
...
+clusterawsadm bootstrap iam create-cloudformation-stack
+```
+The output will give you the policies that have been created for the next step.
+
+## Create a user for the CAPA-Controller
+Create the user
+```
+aws iam create-user --user-name capa-controller
+```
+Attach all the policies from the previous step like so
```
Comment (TheoBrigitte) [inline README.md:62]: In fact we only need the `controllers.cluster-api-provider-aws.sigs.k8s.io` policy.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,71 @@
...
+```
+aws iam attach-user-policy --user-name capa-controller --policy-arn $POLICY_ARN
+```
+
+## Set Credentials
+Create the access key
+```
+aws iam create-access-key --user-name capa-controller
+```
+This key will be used in the CAPA-controller secret.
```
Comment (TheoBrigitte) [inline README.md:72]: The secret looks like this

```yaml
apiVersion: v1
kind: Secret
type: Opaque
metadata:
  name: <some name>
  namespace: <some namespace>
stringData:
  credentials: |-
    [default]
    aws_access_key_id: < AccessKeyId >
    aws_secret_access_key: < SecretAccessKey >
    region: < some region >

```

File: README.md (hunk 1 above)
Comment (anvddriesch) [reply README.md:62]: Nice, that will make it less complicated :)

**Comment by anvddriesch** [comment]
I added the missing steps about the installations repo!

**Pull request merged**